plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -160,19 +161,22 @@ private boolean validateSdk(Path location) {
     } catch (AppEngineException ex) {
       // accept a seemingly invalid location in case the SDK organization
       // has changed and the CloudSdk#validate() code is out of date
+      // todo: does this really make sense? In that case, wouldn't all the
```

Comment:
`appengine-plugins-core` checks for [various path locations](https://github.com/GoogleCloudPlatform/appengine-plugins-core/blob/master/src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java#L69) whereas we just use `gcloud` and `dev_appserver`.